extensions/amp-video/0.1/amp-video.js

```diff
@@ -206,18 +312,25 @@ class AmpVideo extends AMP.BaseElement {
 
   /** @override */
   mutatedAttributesCallback(mutations) {
-    if (!this.video_) {
+    if (!this.isConnected_) {
```

Comment:
hmmm, can `mutatedAttributesCallback` even be called if a node is not connected? /cc @choumx 

I feel like when `mutatedAttributesCallback` is called both `connectedCallback` and `buildCallback` should have already been called.

Reply:
I believe that's right. Maybe assert instead?